core/src/main/java/org/frankframework/pipes/AbstractPipe.java

```diff
@@ -82,7 +83,7 @@
  *
  * @see PipeLineSession
  */
-public abstract class AbstractPipe extends TransactionAttributes implements IPipe, EventThrowing, ApplicationContextAware, IWithParameters {
+public abstract class AbstractPipe extends TransactionAttributes implements IPipe, EventThrowing, ApplicationContextAware, IWithParameters, HasStatistics {
```

Comment:
Adding it here enables us to remove the declaration of this interface at e.g. `MessageSendingPipe` right? Otherwise it is duplicate.

Reply:
I can confirm there is indeed 1 duplicate now.